### ISSUE TITLE
[Bug] `useStorage` hook deletes the value of the `key` in MMKV storage.

### ISSUE THREAD

**Issue by MasterMedo** [ammarahm-ed/react-native-mmkv-storage#123]
**Describe the bug**
Upon the creation of the storage if I enter a value under a key `categories`, and later create a hook `useStorage` on that key, the contents of the storage is lost.

**To Reproduce**
```jsx
import React, { useState } from 'react';
import { Text, View } from 'react-native';
import MMKVStorage, { useMMKVStorage } from 'react-native-mmkv-storage';


const MMKV = new MMKVStorage.Loader().initialize();
MMKV.setArray('categories', ['Hello', 'my', 'friends'])
export const useStorage = (key: string) => {
  // @ts-ignore
  return useMMKVStorage(key, MMKV);
};

export default function App() {
  const [categories, setCategories] = useStorage('categories');
  const [text, onChangeText] = useState<string>('');
  return (
    <View>
      <Text>{(categories || ['not working']).join(' ')}</Text>
    </View>
  );
}
```

**Expected behavior**
I expect the screen to say ''Hello my friends", but it says "not working".

**Platform Information:**
 - OS: iOS
 - React Native Version: https://github.com/expo/react-native/archive/sdk-41.0.0.tar.gz
 - Library Version: ^0.5.9
 - I'm running it with Expo (don't have a Mac to test building it)

**Comment by ammarahm-ed** [comment]
I will try to reproduce it. Have you tried getting the value after setting without a hook by using `getArray` method?

**Comment by MasterMedo** [comment]
Yeah, I tried explicitly  calling `MMKV.setArray` the line after creating the hook, that doesn't work! But I figured that's because of some react flow. I even tried using the hook after creating it, before rendering and that also doesn't work, as expected (because it would re-render into oblivion). But using the hook on button press works as expected. The problem currently is I have to write everywhere `(categories || [])` which is pretty tedious.

EDIT: ups, I misread, I haven't tried `MMKV.getArray` yet, I'll try it.
